### PR TITLE
Reformulate CLI import to make clear the ssh usage

### PR DESCRIPTION
**Pull request by pwalczysko**
Fix issue https://github.com/ome/omero-guides/issues/3 reported by 

@will-moore

### CODE REVIEW

File: docs/import-cli.rst [hunk 1]
```diff
@@ -138,13 +137,29 @@ A user performing an in-place import MUST have:
...
+#. The aim is to import an image from ``/OMERO/in-place-import/FRAP``::
+    
+   $ ls /OMERO/in-place-import/FRAP
+
+#. Activate the virtual environment where ``omero-py`` is installed or add it to ``PATH``. In the example below, the path to the OMERO.server is ``/opt/omero/server``::
+
+   $ export PATH=/opt/omero/server/venv3/bin:$PATH
 
--  Log in again::
+#. Point ``OMERODIR`` to the location where the OMERO server is installed e.g.::
```
Comment (joshmoore) [inline docs/import-cli.rst:150]: Quite soon this won't be necessary (at the cost of a 100 MB download)

File: docs/import-cli.rst [hunk 2]
```diff
@@ -51,60 +51,59 @@ Setup
...
 
-Note: When importing for another user using the CLI, the importer1 does not have to be a member of the target group.
+Note: When importing for another user using the CLI, the ``importer1`` does not have to be a member of the target group.
 
 
 Step-by-step
 ------------
 
-#.  Open a terminal and connect to the server as importer1 using ``ssh``.
+#. If you did not do so already, open a terminal wondow on your local machine and activate the conda environment where your ``omero-py`` is installed (see the Setup section above)::
```
Comment (dominikl) [inline docs/import-cli.rst]:
```suggestion
#. If you did not do so already, open a terminal window on your local machine and activate the conda environment where your ``omero-py`` is installed (see the Setup section above):: 
```

**Comment by dominikl** [comment]
Looks good to me, just one typo.

**Comment by pwalczysko** [comment]
@dominikl thank you, merging

**Pull request merged**